### PR TITLE
i've created get handler and get controller

### PR DESCRIPTION
**Pull request by OsvaldoCorneli**
getTransaction it's done.

### CODE REVIEW

File: src/Controllers/transactionControllers/getTransactionControllers.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const {Transaction} = require('../../db')
+
+const getTransactionController = async(id) =>{
+    console.log("controller",id)
```
Comment (MartinGarciaHervas) [inline src/Controllers/transactionControllers/getTransactionControllers.js]: Eliminame esto pa

**Review by MartinGarciaHervas — CHANGES_REQUESTED** [review]
Eliminame el console.log pa

File: src/Controllers/transactionControllers/getTransactionControllers.js (hunk 1 above)
Comment (OsvaldoCorneli) [reply src/Controllers/transactionControllers/getTransactionControllers.js]: Perfecto me fijo

**Review by MartinGarciaHervas — APPROVED** [review]
Ahora si mi king bro

**Pull request merged**